docs/arcaflow/getting-started.md

```diff
@@ -105,142 +189,488 @@ input:
-    
-    ```yaml title="config.yaml"
-    deployer:
-      type: podman
-      deployment:
-        # Make sure we don't try to pull the image we have locally
-        imagePullPolicy: Never
+    ```bash
+    podman run --rm quay.io/arcalot/arcaflow-plugin-example --json-schema output
+    ```
+=== "Docker"
+    ```bash
+    docker run --rm quay.io/arcalot/arcaflow-plugin-example --json-schema output
     ```
 
-!!! tip
-    Further reading: [Setting up Arcaflow](running/setup.md)
+```json title="example plugin output schema JSON"
+{
+  "$id": "hello-world",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Hello world! outputs",
+  "description": "Says hello :)",
+  "oneof": [
+    {
+      "output_id": {
+        "type": "string",
+        "const": "success"
+      },
+      "output_data": {
+        "type": "object",
+        "properties": {
+          "message": {
+            "type": "string"
+          }
+        },
+        "required": [
+          "message"
+        ],
+        "additionalProperties": false,
+        "dependentRequired": {}
+      }
+    },
+    {
+      "output_id": {
+        "type": "string",
+        "const": "error"
+      },
+      "output_data": {
+        "type": "object",
+        "properties": {
+          "error": {
+            "type": "string"
+          }
+        },
+        "required": [
+          "error"
+        ],
+        "additionalProperties": false,
+        "dependentRequired": {}
+      }
+    }
+  ],
+  "$defs": {
+    "SuccessOutput": {
+      "type": "object",
+      "properties": {
+        "message": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "message"
+      ],
+      "additionalProperties": false,
+      "dependentRequired": {}
+    },
+    "ErrorOutput": {
+      "type": "object",
+      "properties": {
+        "error": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "error"
+      ],
+      "additionalProperties": false,
+      "dependentRequired": {}
+    }
+  }
+}
+```
 
-## Step 7: Run the workflow
+A plugin takes its input as a file, but because it runs as a container, it looks for the input file in the context of the container. This means you either need to bind-mount the input file to the container, or, as in this example, pipe the input value to the plugin's file input.
 
-Finally, let's run our workflow. Make sure you are in the directory where the workflow is located.
+```yaml title="input.yaml"
+name:
+  _type: nickname
+  nick: Arcalot
+```
 
-=== "Linux/MacOS"
+!!! note
+    In order to pipe the input to the container, you must pass the `-i, --interactive` parameter.
 
+=== "Podman"
+    ```bash
+    cat input.yaml | podman run -i --rm quay.io/arcalot/arcaflow-plugin-example -f -
     ```
-    /path/to/arcaflow -input input.yaml -config config.yaml
+=== "Docker"
+    ```bash
+    cat input.yaml | docker run -i --rm quay.io/arcalot/arcaflow-plugin-example -f -
     ```
```

Comment:
This idiom of `cat`'ing into a pipe is dismaying.  The shell is quite capable of redirecting input from a file without having to create an additional process and interprocess pipe.  E.g.,

```bash
podman run -i --rm quay.io/arcalot/arcaflow-plugin-example -f - <input.yaml
```
Yeah, it puts the input at the end of the line, which is unglamorous, but it's not weird.  (Actually, I think you can put the redirection _anywhere_ on the line and the shell will pick it out for you, but that probably _would_ be weird....)

Reply:
You can actually put it first!  😆 
```bash
<input.yaml podman run -i --rm quay.io/arcalot/arcaflow-plugin-example -f -
```

Reply:
I never knew that you could put the redirect first. Fascinating. Weird, but fascinating. However, it does emphasize how we're passing the input, and anyone familiar with shell syntax will recognize what it does even if it makes them pop over to a terminal window to check "wait, that really works?", like I just did ... 😆

Reply:
I can't fathom why this matters in the least. We are giving examples of how to run these commands with the needed data passing, using I would argue one of the most approachable if not technically the "simplest" methods. I don't think we need to provide a lesson in shell redirection here.